resources/web/docs_js/components/console_widget.jsx

```diff
@@ -33,23 +33,25 @@ export class _ConsoleForm extends Component {
 
     return (
       <form>
+         <p><strong>Want to use the Kibana Console?</strong></p>
         <label for="url">{props.langStrings(props.url_label)}</label>
         <input
           id="url"
           type="text"
           value={getValueFromState('url')}
           onInput={linkState(this, getFieldName('url'))}
         />
-
-        <label for="curl_host">curl {props.langStrings('host')}</label>
+        <p></p>
+        <p><strong>Want to use curl? (basic auth)</strong></p>
```

Comment:
-OR-

Secondary header: `Local deployment`

Below the Elasticsearch username field:
`For information on Kibana settings, go to Configure Kibana [https://www.elastic.co/guide/en/kibana/master/settings.html].`

Move the `Save` and `Reset` buttons to the bottom of the window.

Reply:
You can use both Console and for cloud and local, and the same is technically true for curl except the curl settings options are incomplete for a cloud deployment requiring API keys. So just wondering if it makes sense to distinguish by cloud/local rather than console/curl.